### PR TITLE
Hello Microverse setup

### PR DESCRIPTION
**Pull request by maroun7**
Add an HTML file called index.html
--Text that say "Hello Microverse!"

Add a CSS file called style.css
--Linked to index.html
--Change the font color to blue

Add a README.md file

Add a .gitignore file
--Insert a test.md file

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (descholar-ceo) [inline test.md:1]: - [ ] As it is said, this file should be ignored by git, this means that it shouldn't appear here in your repo, removing it, you can use `git rm -f test.md` and writing it on gitignore, you should remove the starting slash, and then commit changes

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: 
## Hi @maroun7  :handshake: :handshake: :handshake:  , 

Good job so far!
We are so excited to have you here at Microverse, with your first project here!!! :smile: :smile: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :high_brightness: :heavy_check_mark: 
:+1: Github flow was correctly :heavy_check_mark:

## Required Changes ♻️
- [ ] Linters are not setup, this setup is mandatory to have it on your repository, because it will help you to write more professional codes, to set it up, you can refer to [this guide](https://github.com/microverseinc/linters-config/tree/master/html-css-js)
- [ ] This Pull Request should have a descriptive title, which shows what kind of changes introduced while working on it, try something like: `Setup project`, `Add initial project files`, ... And also it should have a short description which contains briefly what has been introduced on this Pull request
- [ ] You should customize the readme file and add information relevant to your current project instead of letting the default microverse template

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @maroun7    ! :wave: :wave:

#### You have done really well in implementing thepreviously  required changes.
- You used proper GitHub Flow,
- Your linter checks are all passing,
- Your README is descriptive. 👍🏾 

 However, there are changes required for you to make in order for this project to be approved. 
- - -

### **Recommended** ♻️

- [ ] As requested by the previous reviewer, you should have a descriptive PR message.  It's essential that your repository is professional, as this is one of the things that potential colleagues will look at when recruiting team members. 



_See the inline comments below for more required changes_

- - - 
**Note that you still have 1 more review as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @maroun7! 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

doing good


Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**